spec/cypress/integration/admin/administrators/delete-administrator-user.js

```diff
@@ -0,0 +1,18 @@
+describe("Admin user deleting another admin user", () => {
+  const adminUserName = "Emma Dow";
+
+  beforeEach(() => {
+    cy.login("admin");
+  });
+
+  it("should allow deleting user", () => {
+    cy.appScenario("/admin/administrators/create_admin_user");
+    cy.visit("/admin/administrators");
+    cy.get("main").should("contain", adminUserName);
+    cy.get('.cypress-test-edit-admin-link').contains(adminUserName).click();
+    cy.get('.cypress-test-delete-button').click()
+    cy.get('.cypress-test-delete-button').click()
```

Comment:
This isn't especially readable to me - is it clicking a delete button and then a confirm deletion button, or two delete buttons?

I think adding an assertion between them would help (e.g., if it's going to a confirm deletion button, test that it's on the confirm page)

Reply:
Added additional check inbetween actions